bin/sozo/src/ops/events.rs

```diff
@@ -40,124 +39,455 @@ pub async fn execute(
+                    }),
+                },
+                CompositeInner {
+                    index: 3,
+                    name: "u16".to_string(),
+                    kind: CompositeInnerKind::Data,
+                    token: Token::CoreBasic(CoreBasic {
+                        type_path: "core::integer::u16".to_string(),
+                    }),
+                },
+                CompositeInner {
+                    index: 4,
+                    name: "u32".to_string(),
+                    kind: CompositeInnerKind::Data,
+                    token: Token::CoreBasic(CoreBasic {
+                        type_path: "core::integer::u32".to_string(),
+                    }),
+                },
+                CompositeInner {
+                    index: 5,
+                    name: "u64".to_string(),
+                    kind: CompositeInnerKind::Data,
+                    token: Token::CoreBasic(CoreBasic {
+                        type_path: "core::integer::u64".to_string(),
+                    }),
+                },
+                CompositeInner {
+                    index: 6,
+                    name: "u128".to_string(),
+                    kind: CompositeInnerKind::Data,
+                    token: Token::CoreBasic(CoreBasic {
+                        type_path: "core::integer::u128".to_string(),
+                    }),
+                },
+                CompositeInner {
+                    index: 7,
+                    name: "usize".to_string(),
+                    kind: CompositeInnerKind::Data,
+                    token: Token::CoreBasic(CoreBasic {
+                        type_path: "core::integer::usize".to_string(),
+                    }),
+                },
+                CompositeInner {
+                    index: 8,
+                    name: "class_hash".to_string(),
+                    kind: CompositeInnerKind::Data,
+                    token: Token::CoreBasic(CoreBasic { type_path: "core::ClassHash".to_string() }),
+                },
+                CompositeInner {
+                    index: 9,
+                    name: "contract_address".to_string(),
+                    kind: CompositeInnerKind::Data,
+                    token: Token::CoreBasic(CoreBasic {
+                        type_path: "core::ContractAddress".to_string(),
+                    }),
+                },
+            ],
+            generic_args: vec![],
+            r#type: CompositeType::Struct,
+            is_event: true,
+            alias: None,
+        };
+        let tokenized_composite = Token::Composite(composite);
+
+        let mut events_map = HashMap::new();
+        events_map
+            .insert(starknet_keccak("TestEvent".as_bytes()).to_string(), vec![tokenized_composite]);
+
+        let event = EmittedEvent {
+            keys: vec![starknet_keccak("TestEvent".as_bytes())],
+            data: vec![
+                FieldElement::from_hex_be("0x5465737431").expect("Invalid hex"),
```

Comment:
In testing you can simplify when you know that the input is expected to be correct. :+1: 
```suggestion
                FieldElement::from_hex_be("0x5465737431").unwrap(),
```